### PR TITLE
update ai.md

### PR DESCRIPTION
**Pull request by gam81**

### CODE REVIEW

**Comment by gam81** [comment]
Update ai.md

File: adv_topics/ai.md [hunk 1]
```diff
@@ -24,7 +25,37 @@
...
+    - Machine learning
+       - Statistical learning
+* Planned implementation
+    - Enemy behavior
+       - Unintelligent horde combat
+       - Gimmicky triggers and spawns
+    - Boss behavior
+    - Group vs solo attacking
+    - Varied pathing
+* Examples of Metroidvania A.I.
```
Comment (nfarnan) [inline adv_topics/ai.md:49]: Intro group already presented examples of game AI, you shouldn't revisit that here. I could see a max 1 slide listing the influences on your game AI before you describe your implementation.

File: adv_topics/ai.md [hunk 2]
```diff
@@ -24,7 +25,37 @@
     - Halo, Fear, Arkham Trilogy, Shadow of War, Alien: Isolation, Sims
 
 ## Presentation II (Oct. 26):
-* 
+* Review of last presentation
```
Comment (nfarnan) [inline adv_topics/ai.md:28]: Don't need to review. Students will have the recording and the slides. Cut this

File: adv_topics/ai.md [hunk 3]
```diff
@@ -24,7 +25,37 @@
...
+    - Definitions
+    - Uses
+    - examples
+* Concepts to emulate
+    - Hp gates
+    - Reactionary behavior and adaptation
+    - Modification via difficulty settings 
+    - Arbitrary boss resources to prevent predictable or unreactable move-spamming
+* Dynamic principles
+    - Bayes equation for spam filters and how filtering developed
```
Comment (nfarnan) [inline adv_topics/ai.md:38]: How will you be using Bayes filtering in your game?

File: adv_topics/ai.md [hunk 4]
```diff
@@ -24,7 +25,37 @@
...
+       - Naive learning
+    - Machine learning
```
Comment (nfarnan) [inline adv_topics/ai.md:40]: How will you be using ML in your game?

**Review by nfarnan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**